### DISCUSSION TITLE
阿里网盘的内容不删除

### DISCUSSION THREAD

**Discussion by Elvin10311** [monlor/docker-xiaoya#186] [Q&A]
![微信截图_20240708215612](https://github.com/monlor/docker-xiaoya/assets/19567916/9e5f1a29-e194-4588-b490-7fb0a08cc60b)
阿里的网盘不自动删除。。

**Comment by monlor** [comment]
更新一下，中间有版本跟小雅不兼容，定时任务没了

**Reply by Elvin10311** [reply]
我今天晚上刚更新的，还用整体覆盖更新么？token也需要重新提取么 ？
